src/components/MergeArticleList.tsx

```diff
@@ -7,10 +7,20 @@ const Container = styled.div`
 `
 
 const reads: Array<CardListItem> = [
+   {
+    title: "Technical Aspects of Ethereum Merge",
+    description: "Alchemy",
+    link: "https://www.alchemy.com/overviews/the-ethereum-merge",
```

Comment:
My 2 gwei: looks like we already cover pretty much all this content on: https://ethereum.org/en/upgrades/merge/